src/app/app.component.css

```diff
@@ -1,7 +1,5 @@
-h1 {
-  font-size: 1.2em;
-  color: #999;
-  margin-bottom: 0;
+nav {
+  margin-top: 1rem;
```

Comment:
I hadn't seen "rem" units before, cool. That seems potentially more sane than specifying sizes relative to whatever the inherited/parent's style happens to be.